README.md

```diff
@@ -32,13 +32,17 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+* _John_: I managed enterprise Java servers for over a decade without really ever learning Java itself.  I could read it well enough, but never really learned all the conventions.  All the coding I ever did was functional (Perl) or procedural (PHP and Python), so OO always felt "wrong" to my brain.  I kept saying I was going to learn it and when Nationwide was offering to pay for me to learn it, I figure this was the impetus to really so it.  We maintain a wrapper of sorts on my team that handles MFA traffic for the enterprise and it's in Java, I figure this will allow me to dive into it aside from just maintaining the infrastructure.
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+* _John_: The best place to be is where I can find someone like me, someone who's never met a stranger.  I prefer places with deep history.  Heidelberg is pretty amazing, New Orleans is deep.  Bucket list, Prague.  
 
 Q3: Name your favorite member of [The Beatles](https://en.wikipedia.org/wiki/The_Beatles), and your favorite Beatles song.  You do know at least one Beatles song, right?  If you are a Beatles fan, feel free to give me your top 5 or so.
+* _John_: Favorite: _Yesterday_. When I was a kid, my dad and I would go fishing on Lake Erie.  We had this small FM radio that was shaped like a Coke can.  There was a long "no wake" channel from where we would put the boat in to where we could take off for a good fishing spot.  We would putter along that channel for a good 4-5 minutes, usually me driving while dad would get the fishing gear ready.  The song I most associate with hearing during those trips was Yesterday.
+ Next tops would be: _Hey Jude_, _Norwegian Wood_, _Blue Jay Way_ and _Being for the Benefit of Mr. Kite_.
```

Comment:
Nice.  Do you have a favorite band member?  Please add your answer to your interview response and update your pull request.